libr/anal/meson.build

```diff
@@ -112,15 +112,14 @@ r_anal_sources = [
   'p/anal_tricore.c',
   'p/anal_vax.c',
   'arch/vax/vax-dis.c',
-  'p/anal_wasm.c',
+  'p/anal_ws.c',
```

Comment:
uh? is this branch rebased?

Reply:
yeah... I must of messed that up on merge conflicts. Thanks for checking it. I will get it fixed.